vic/drivers/classic/src/read_veglib.c

```diff
@@ -163,10 +163,10 @@ read_veglib(FILE   *veglib,
             /* Carbon-cycling parameters */
             if (options.VEGLIB_PHOTO) {
                 fscanf(veglib, "%s", tmpstr); /* photosynthetic pathway */
-                if (!strcmp(tmpstr, "C3")) {
+                if (!strcmp(tmpstr, "0") || !strcmp(tmpstr, "C3")) {
                     temp[i].Ctype = PHOTO_C3;
                 }
-                else if (!strcmp(tmpstr, "C4")) {
+                else if (!strcmp(tmpstr, "1") || !strcmp(tmpstr, "C4")) {
```

Comment:
OK, will do.  Except that it's the strings "C3" and "C4" that I'm deprecating.  And the reason for that is that I had trouble figuring out how to get them to work correctly in tonic - I tried for hours - and finally relented and made them ints.  So, tonic can read either of them (string or int), but will convert to the 0/1 values for writing to netcdf.  So, VIC is reading them as ints in image mode.